src/clinical/service-worker-thread/tasks.ts

```diff
@@ -140,14 +147,14 @@ const mapEntityDocuments = (
     records = records.slice(first, last);
   }
 
-  const completionRecords =
+  const CompletionDisplayRecords =
```

Comment:
camelCase please

Reply:
Oops find and replace gone wrong
Updated